manas_cafa5/protein.py

```diff
@@ -95,6 +96,23 @@ def get_children(self, term_type, graph, max_distance):
             for term_id in term_set
         ]
 
+    def get_parents(self, term_type, graph):
+        term_set = reduce(
+            lambda terms, term: terms.union(
+                networkx.ancestors(graph, term['id'])
+            ),
+            self.get_terms(term_type),
```

Comment:
Here we do not want the actual terms: we will use this function in order to compute candidate terms, the actual terms for a protein are not candidates, they are already assigned to the protein then they will give us no score in the predictions. We just want the ancestors.

Also, will be good to have a parameter `distance` like the descendants function where we compute the ancestors up to any specific degree.